mb-solr/src/main/resources/oxml.xml

```diff
@@ -261,6 +261,7 @@
         <java-type name="ReleaseGroup">
             <java-attributes>
                 <xml-element java-attribute="secondaryTypeList" xml-path="."/>
+	            <xml-element java-attribute="primaryType" xml-path="." />
```

Comment:
It's a minor point, but several of the edits in this file seem to use different indentation settings (tabs vs spaces, most likely)

Reply:
Right, I thought the file already has inconsistent indentation. So, I opened a different PR #42 for cleaning up this file and fixing the formatting.